tsi-typescript.el

```diff
@@ -427,15 +428,11 @@ where the current line is empty."
                (and (eq current-type 'arrow_function)
                     (eq parent-type 'arguments))
                
-               (and
-                (eq current-type 'object)
-                (or (eq parent-type 'return_statement)
```

Comment:
Not related to the fixes in this MR, but this block was no longer needed, after introducing the the expression below using `tsi-typescript--double-nestable-types` in #26.